[fix #5] [mozmeao/infra#1104] Clean up oregon-b stage yaml.

jpetto opened the pull request:
- rename svc.yaml to web-svc.yaml

jpetto committed:
[fix #5] [mozmeao/infra#1104] Clean up oregon-b stage yaml.

- rename svc.yaml to web-svc.yaml

jpetto committed:
update image hash and add k8s namespace/podname

jgmize commented on oregon-b/basket-stage/donateworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-stage-v110-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-stage/donateworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-stage-v110-env
+          value: 3
```
```suggestion
          value: "3"
```

jgmize commented on oregon-b/basket-stage/donateworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-stage-v110-env
+          value: 2
```
```suggestion
          value: "2"
```

jgmize commented on oregon-b/basket-stage/donateworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-              key: wsgi-num-workers
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-stage/donateworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-stage/fxaeventsworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-stage-v110-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-stage/fxaeventsworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-stage-v110-env
+          value: 3
```
```suggestion
          value: "3"
```

jgmize commented on oregon-b/basket-stage/fxaeventsworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-stage-v110-env
+          value: 2
```
```suggestion
          value: "2"
```

jgmize commented on oregon-b/basket-stage/fxaeventsworker-deploy.yaml:
```diff
@@ -48,291 +37,188 @@ spec:
-              key: wsgi-num-workers
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-stage/ns.yaml:
```diff
@@ -1,13 +1,7 @@
-  resourceVersion: "1120960"
-  selfLink: /api/v1/namespaces/basket-stage
-  uid: 41727e2f-0153-11e8-8c8b-0656fabf84c0
 spec:
```
Please remove the spec & status sections

jgmize commented on oregon-b/basket-stage/web-deploy.yaml:
```diff
@@ -48,291 +38,188 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-stage-v110-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-stage/web-deploy.yaml:
```diff
@@ -48,291 +38,188 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-stage-v110-env
+          value: 3
```
```suggestion
          value: "3"
```

jgmize commented on oregon-b/basket-stage/web-deploy.yaml:
```diff
@@ -48,291 +38,188 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-stage-v110-env
+          value: 2
```
```suggestion
          value: "2"
```

jgmize commented on oregon-b/basket-stage/web-deploy.yaml:
```diff
@@ -48,291 +38,188 @@ spec:
-              key: wsgi-num-workers
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-stage/web-deploy.yaml:
```diff
@@ -48,291 +38,188 @@ spec:
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-b/basket-stage/web-svc.yaml:
```diff
@@ -0,0 +1,23 @@
+    type: web
+  sessionAffinity: None
+  type: NodePort
+status:
```
Please remove the status section

jgmize commented on oregon-b/basket-stage/worker-deploy.yaml:
```diff
@@ -48,291 +40,188 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-stage-v110-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-b/basket-stage/worker-deploy.yaml:
```diff
@@ -48,291 +40,188 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-stage-v110-env
+          value: 3
```
```suggestion
          value: "3"
```

jgmize commented on oregon-b/basket-stage/worker-deploy.yaml:
```diff
@@ -48,291 +40,188 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-stage-v110-env
+          value: 2
```
```suggestion
          value: "2"
```

jgmize commented on oregon-b/basket-stage/worker-deploy.yaml:
```diff
@@ -48,291 +40,188 @@ spec:
-              key: wsgi-num-workers
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-b/basket-stage/worker-deploy.yaml:
```diff
@@ -48,291 +40,188 @@ spec:
-              name: basket-stage-v110-env
-        image: 127.0.0.1:5555/basket-stage:v110
+          value: 4
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jpetto committed:
Review fixes:

- remove superfluous data from ns.yaml
- quote numeric values in env vars
- use full image hash
